python/tk_multi_launchapp/software_entity_launcher.py

```diff
@@ -228,10 +224,9 @@ def _get_sg_software_entities(self):
         # Now Group and User restrictions. Always retrieve Software entities
         # that have no Group or User restrictions.
         current_user = self._tk_app.context.user
-        user_group_filters = [
-            ["sg_user_restrictions", "is", None],
-            ["sg_group_restrictions", "is", None],
-        ]
+
+        # No user restriction filter.
+        user_group_filter = ["user_restrictions", "is", None]
```

Comment:
Actually, user_restrictions is now both users and groups. See [here](https://github.com/shotgunsoftware/tk-multi-launchapp/blob/72429ae66ab3432a19c6bb590234ea27d55b0363/python/tk_multi_launchapp/software_entity_launcher.py#L241-L245)